pandas/tests/io/test_clipboard.py

```diff
@@ -123,6 +124,15 @@ def test_read_clipboard_infer_excel(self):
         exp = pd.read_clipboard(**clip_kwargs)
 
         tm.assert_frame_equal(res, exp)
+        
+    def test_excel_clipboard_format(self):
+        for dt in self.data_types:
+            for sep in ['\t', None]:
+                data = self.data[dt]
+                data.to_clipboard(excel=True, sep=sep)
```

Comment:
Can you also test with `excel=False`?  In <=0.22 it would also be tab delimited

Reply:
You mean to test `data.to_clipboard(excel=False, sep=sep)`?  I don't think that was (or should be) tab delimited.  Do you mean `excel=None`?

Reply:
Sorry, right, I meant to the default `df.to_clipboard()`  (sep=None, excel=None)

```
# pandas 0.22
In [72]: df = pd.DataFrame({'a': [1, 3], 'b': [4, 5]})

In [73]: df.to_clipboard()

# <paste>
In [74]: res = """	a	b
    ...: 0	1	4
    ...: 1	3	5"""

In [75]: res.count('\t')
Out[75]: 6

Reply:
Good point.  Should be added